proto/pgctld.proto

```diff
@@ -0,0 +1,211 @@
+// Copyright 2025 The Multigres Authors
```

Comment:
should we name thiss  file `pgctldservice.proto`? So we keep them seperate from the messages